src/routes/prometheus.js

```diff
@@ -0,0 +1,53 @@
+import { Router } from "express";
+
+import client, { Counter, Summary } from "prom-client";
+
+import timeResponse from "response-time";
+
+const SPECIAL_ENDPOINTS = ["/api-docs", "/metrics", "/ready", "/live"];
+
+const router = Router();
+
+client.register.clear();
+const numOfRequests = new Counter({
+  name: "numOfRequests",
+  help: "Number of requests made",
+  labelNames: ["method"],
+});
+
+const pathsTaken = new Counter({
+  name: "pathsTaken",
+  help: "Paths taken in the app",
+  labelNames: ["path"],
+});
+
+const responses = new Summary({
+  name: "responses",
+  help: "Response time in millis",
+  labelNames: ["method", "path", "status"],
+});
+
+router.use((req, _, next) => {
+  const currentEndpoint = "/" + req.originalUrl.split("/")[1];
+  if (!SPECIAL_ENDPOINTS.includes(currentEndpoint)) {
```

Comment:
I think instead of ignoring the list of special endpoints it makes more sense to just do `router.use("/api/v1/x", ...)

Reply:
scratch that leave this as is (minus special endpoints) and see comment above in app.js